REFERENCE.md

```
@@ -6070,6 +6070,363 @@ print(f'Magic Link: {response[0].magic_link}')
 
+</dd>
+</dl>
+</details>
+
+## Custom Providers
+
+<details><summary><code>client.actions.providers.<a href="https://github.com/scalekit-inc/scalekit-sdk-python/blob/main/scalekit/actions/actions.py">create_custom_provider</a>(request) -> CreateCustomProviderResponse</code></summary>
+<dl>
+<dd>
+
+#### 📝 Description
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)